src/app/dataValidator/dataValidator.js

```diff
@@ -44,11 +82,11 @@ const validateData = data => {
   log('\nValidation complete!');
 };
 
-// invoke the validator for WIP purposes
 const data = require('../../../data/test/scenario-01.json');
 
 validateData(data);
 
 module.exports.validateNode = validateNode;
+module.exports.validateProperties = validateProperties;
```

Comment:
They need to be exported to be tested. The longer I've looked at this I think I should use import and export, which then means babel needs to transpile the code before it can be executed as a node command.

Reply:
This has since changed due to the code restructuring